internal/website/gatherexamples/gatherexamples.go

```diff
@@ -209,14 +226,21 @@ rewrite:
 			sb.WriteString("return err")
 			sb.WriteByte('\n')
 		default:
-			if !strings.HasPrefix(line[start:], signifierCommentPrefix) {
+			const importBlankPrefix = "// import _ "
```

Comment:
Add a comment that explains the logic in `default:` on a high level

Reply:
The comment is effectively a continuation of the earlier "check if needs rewrites", so I realized that the better thing to do was flatten the `switch` statement. Hopefully this makes it easier to read.